pkgs/development/tools/misc/prelink/default.nix

```diff
@@ -1,22 +1,46 @@
-{ lib, stdenv, fetchurl, libelf }:
+{ stdenv
+, lib
+, fetchgit
+, autoreconfHook
+, libelf
+, libiberty
+}:
 
 stdenv.mkDerivation rec {
   pname = "prelink";
-  version = "20130503";
+  version = "unstable-2019-06-24";
+
+  src = fetchgit {
+    url = "https://git.yoctoproject.org/git/prelink-cross";
+    branchName = "cross_prelink";
+    rev = "f9975537dbfd9ade0fc813bd5cf5fcbe41753a37";
+    sha256 = "sha256-O9/oZooLRyUBBZX3SFcB6LFMmi2vQqkUlqtZnrq5oZc=";
+  };
+
+  strictDeps = true;
+
+  configurePlatforms = [ "build" "host" ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
 
   buildInputs = [
-    libelf stdenv.cc.libc (lib.getOutput "static" stdenv.cc.libc)
+    stdenv.cc.libc
+    libelf
+    libiberty
   ];
 
-  src = fetchurl {
-    url = "https://people.redhat.com/jakub/prelink/prelink-${version}.tar.bz2";
-    sha256 = "1w20f6ilqrz8ca51qhrn1n13h7q1r34k09g33d6l2vwvbrhcffb3";
-  };
+  # PASS 36 FAIL 10
+  # ld: cannot find reloc2lib2.so: No such file or directory
+  # cp: cannot create regular file 'deps1.tree/lib/libstdc++.so.6.0.29.orig': Permission denied
+  doCheck = false;
```

Comment:
Remove some testcases so we at least have some coverage?

Reply:
done